docs/wiki/[variables]-relaxed_management_group_subscription_association.md

```diff
@@ -0,0 +1,29 @@
+<!-- markdownlint-disable first-line-h1 -->
```

Comment:
We should probably add a note to this page stating that the platform Subscriptions should always be associated to their respective Management Group by this module due to the other dependencies on these inputs.